packages/fluent-theme/src/fluent/styles/Dialog.styles.ts

```diff
@@ -4,7 +4,7 @@ import {
   IDialogFooterStyleProps,
   IDialogFooterStyles
 } from 'office-ui-fabric-react/lib/Dialog';
```

Comment:
Problem runs very deep here - the problem is that we have all of these imports from OUFR/lib/components
@Vitalius1 - maybe you can take a crack at address this?
@dzearing - should OUFR itself be a dependency for fluent-theme at all?

Reply:
I don't see how to get around having OUFR as a dependency for theming packages. These styles are targeting specific component props and styles and need component typings.

Reply:
If that is the case, then we are actually implying that by using fluent theme, then you cannot do SSR. That's a TERRIBLE trade off.

Reply:
To be clear, I wasn't commenting on HOW this is imported (as I'm not sure the options here), but only that theming packages will need component typings unless we forego all type safety (or making a typings package?)

Reply:
Other hacks one can do would be to rewrite "require" like with "rewire" package or something... REALLY seems extreme to me.

Reply:
When I spoke to @dzearing about this, he opted to optimize for AMD consumers (given that we know of major partners on AMD consuming Fabric) over CommonJS consumers...

For CommonJS consumers, I think David recommended they do some workaround since we can't solve it for both AMD/CommonJS at the same time. One possible workaround I can think of is that they would just have to use something like babel/TS to transpile the ESM into CJS.

Reply:
And.. yeah, this inconsistency in our imports is a bit of a mess (including how we special optimize for AMD). From @dzearing's POV, the long term fix is to investigate breaking apart the OUFR package (i.e. each component is its own package import).